mlflow/server/__init__.py

```diff
@@ -34,6 +35,9 @@
 SERVE_ARTIFACTS_ENV_VAR = "_MLFLOW_SERVER_SERVE_ARTIFACTS"
 ARTIFACTS_ONLY_ENV_VAR = "_MLFLOW_SERVER_ARTIFACTS_ONLY"
 
+GATEWAY_HOST_VAR = "MLFLOW_GATEWAY_HOST"
+GATEWAY_PORT_VAR = "MLFLOW_GATEWAY_PORT"
```

Comment:
Let's add these is `mlflow/environment_variablees.py`.

Reply:
@harupy I see that in `mlflow/environment_variables.py` there is already a `MLFLOW_GATEWAY_URI` var. Can we re-use this?

Reply:
Sounds like a reasonable approach. @dbczumar wdyt?